applications/fishing-map/src/features/app/app.selectors.ts

```diff
@@ -40,16 +38,6 @@ export const selectViewport = createSelector(
   }
 )
 
-export const selectTimeRange = createSelector(
```

Comment:
restored this as well as using DEFAULT_TIME_RANGE was causing rendering and requesting 4wings tiles twice